example_configs/dolibarr.md

```diff
@@ -0,0 +1,88 @@
+# Configuration pour Dolibarr
+
+## This example will help you to create user in dolibarr from your users in your lldap server from a specific group and to login with the password from the lldap server.
+
+## to connect ldap->dolibarr 
+
+Install module LDAP from Home -> Modules/Applications
+Go to the configuration of this module and fill it like this:
+
+
+Users and groups synchronization: LDAP -> Dolibarr
+Contacts' synchronization: No
+Type: OpenLdap
+Version: Version 3
+Primary server: ldap://example.com
+Secondary server: Empty
+Server port: port 389
+Server DN: dc=example,dc=com
+Use TLS:  No
+Administrator DN: cn=admin,ou=people,dc=example,dc=com
+Administrator password: secret
+
+Click on modify then "TEST LDAP CONNECTION". 
+You should get this result on the bottom:
+```
+ TCP connect to LDAP server successful (Server=ldap://example.com, Port=389)
+Connect/Authenticate to LDAP server successful (Server=ldap://example.com, Port=389, Admin=cn=admin,ou=people,dc=example,dc=com, Password=**********)
+LDAP server configured for version 3
+```
+
+And two new tabs will appear on the top:
+Users and Groups
+
+We will use only Users in this example to get the users we want to import.
+The tab Groups would be to import groups.
+
+Click on the Users tab and fill it like this:
+Users' DN: ou=people,dc=example,dc=com
+List of objectClass: person
+Search filter: memberOf=cn=yournamegroup,ou=groups,dc=example,dc=com
```

Comment:
Maybe add that if they don't have a group, they can just use the `uid=*` filter

Reply:
In this case we could just leave the filter empty ?

Reply:
You can give it a try, but I think LDAP expects a valid filter there. If we can leave it empty, that's even better.

Reply:
Nit: search filter between ``

Reply:
it works without filter